core/src/main/java/jenkins/model/item_category/StandaloneProjectsCategory.java

```diff
@@ -0,0 +1,33 @@
+package jenkins.model.item_category;
+
+import hudson.Extension;
+import jenkins.model.Messages;
+
+/**
+ * A generic {@link ItemCategory}
+ */
+@Extension(ordinal = Integer.MIN_VALUE + 1)
```

Comment:
:ant: Not leaving yourself much room for negotiation there, if someone later wants a category to appear before this one. Better to use, say, -100. (`MIN_VALUE` is appropriate for `uncategorized` since it has a special status.)